src/ExtensionDefinitionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Interop\Container;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * An extension optionally implements this interface to reflect it's dependencies.
+ */
+interface ExtensionDefinitionInterface extends ServiceDefinitionInterface
+{
+    /**
+     * Extends a given service/entry, using the given container to resolve dependencies.
```

Comment:
```suggestion
     * Extends a given service, using the given container to resolve dependencies.
```

Reply:
If it must be one or the other, I prefer "entry" - see above. (I put "service/entry" in a few places for clarity, but yeah..)